plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -194,4 +196,19 @@ public static String imageRef(IBuild build) {
 		return "";
 
 	}
+	
+	/**
+	 * Find the collection of pods for the given deployment config
+	 * @param deploymentConfig
+	 * @param pods
+	 * @return
+	 */
+	public static Collection<IPod> getPodsForDeploymentConfig(IDeploymentConfig deploymentConfig) {
```

Comment:
Similiar to my comment above is this will match only if the annotation is there but a better approach would probably be to try and match the dc.selector to the pod.